include/ncengine/asset/AssetViews.h

```diff
@@ -0,0 +1,121 @@
+#pragma once
```

Comment:
Basically a copy paste from Assets.h, except for new `Font` and `FontView` types.